### DISCUSSION TITLE
Admin user password

### DISCUSSION THREAD

**Discussion by ardaozerten** [DeviceFarmer/stf#481] [Q&A]
Hello everyone, how can I learn about the password of admin user "admistrator@fakedomain.com"?

Also how can I change privilege of other users?

**Comment by denis99999** [comment]
@ardaozerten ,
Look at `README.md` in `Running` section, there is no password but only name and email for mock authentication, only one user can exist with `admin` privileges, currently if you want to modify the `admin` user you must recreate your database but soon it will be possible to do this operation without recreate the database.
Regards.
